tests/e2e/specs/web/components/searchbar.spec.js

```diff
@@ -0,0 +1,12 @@
+module.exports = {
```

Comment:
As far as I know e2e test development is QA's responsibility and we dont need to transfer this file

Reply:
done